ci-operator/config/openshift/hive/openshift-hive-master.yaml

```diff
@@ -218,6 +222,27 @@ tests:
         requests:
           cpu: 100m
     workflow: ipi-azure
+- as: e2e-vsphere
+  run_if_changed: vsphere
+  steps:
+    cluster_profile: vsphere-2
```

Comment:
This cluster profile is incorrect

The only cluster_profile that should be used for an CI jobs now is `vsphere-elastic` only - unless you are running jobs in devqe - which should only be a very special case.